### PR TITLE
Lesson 1.2

### PR DESCRIPTION
**Pull request by hvd28**

### CODE REVIEW

File: src/AddToDoForm.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+function AddToDoForm() {
+    return (
+<form> 
+    <label htmlFor="toDoTitle">Need to add a task?</label>
+    <br></br>
+    <input id="toDoTitle">
```
Comment (alenamedved) [inline src/AddToDoForm.js:8]: input can have a type='text'

File: src/AddToDoForm.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    </input>
+    <button>Add</button>
```
Comment (alenamedved) [inline src/AddToDoForm.js:10]: button has a type='submit'

**Review by alenamedved — COMMENTED** [review]
great job on this task

File: src/AddToDoForm.js (hunk 1 above)
Comment (hvd28) [reply src/AddToDoForm.js:8]: Thanks for the feedback! Just made these adjustments.

**Pull request merged**